circuit-breaker/fuse/common.yaml

```diff
@@ -0,0 +1,5 @@
+name: Spring Boot Circuit Breaker Example
```

Comment:
Is there a risk of confusion with RHOAR Spring Boot?

Reply:
Yes.